src/pages/get-eth.tsx

```diff
@@ -368,7 +368,7 @@ const GetEthPage = ({
           </Text>
           <Text>{t("page-get-eth-dexs-desc-3")}</Text>
           <Text>{t("page-get-eth-need-wallet")}</Text>
-          <ButtonLink href="/wallets/find-wallet/">
+          <ButtonLink href="/wallets/find-wallet/" mb={8}>
```

Comment:
Revert this. This is not related to this issue in question.

```suggestion
          <ButtonLink href="/wallets/find-wallet/">
```

Nice catch though! Please create a new PR for this hotfix. 😄